pkg/cluster/cluster.go

```diff
@@ -68,6 +68,8 @@ func (c *Cluster) Start(ctx context.Context) (<-chan struct{}, error) {
 		c.config.Runtime.EtcdConfig.Endpoints = strings.Split(c.config.Datastore.Endpoint, ",")
 		c.config.Runtime.EtcdConfig.TLSConfig = c.config.Datastore.BackendTLSConfig
 
+		go etcd.RemoveEtcdStatusCondition(c.config.Runtime)
```

Comment:
I think this should be done in the same place that we remove the etcd labels and annotations from non-etcd nodes:
https://github.com/k3s-io/k3s/blob/3190a5faa28d7a0d428c756d67adcab7eb11e6a5/pkg/etcd/metadata_controller.go#L111-L113